### PR TITLE
added support for multiple dotnet versions and port override

### PR DESCRIPTION
**Pull request by jldeen**
- Adds support for .NET Version override (support more than .NET 5.0 I.E 6.0)
- Adds support for PORT override via `--urls` flag

Signed-off-by: jldeen <jessicadeen@me.com>

### CODE REVIEW

File: builders/csharp/Dockerfile [hunk 1]
```diff
@@ -11,11 +11,11 @@ RUN sed -n 's:.*<AssemblyName>\(.*\)</AssemblyName>.*:\1:p' *.csproj > __assembl
 RUN if [ ! -s __assemblyname ]; then filename=$(ls *.csproj); echo ${filename%.*} > __assemblyname; fi
 
 # Stage 2
-FROM mcr.microsoft.com/dotnet/aspnet:5.0
+FROM mcr.microsoft.com/dotnet/aspnet:{{VERSION}}
```
Comment (imiller31) [inline builders/csharp/Dockerfile:14]: Does dotnet sdk version directly map to the tag?

**Comment by imiller31** [comment]
You'll need to have add the language variables to these files `https://github.com/Azure/draft/tree/main/test/integration/csharp` in order to fix the failing integration tests.

Then run, `make generate-integrations`

File: builders/csharp/Dockerfile (hunk 1 above)
Comment (jldeen) [reply builders/csharp/Dockerfile:14]: yes

**Comment by jldeen** [comment]
@imiller31 I believe I resolved the failing integration tests, can you take a look and see if there's anything else you need from me? Thanks!

**Review by imiller31 — APPROVED** [review]
LGTM

**Comment by imiller31** [comment]
@jldeen looks like you'll need to set up commit signing -- we have a requirement to only merge verified commit.

[Here's a guide on how to set it up](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

**Pull request merged**